api/_lighthouse_ide_helper.php

```diff
@@ -1,10 +1,9 @@
-<?php
+<?php declare(strict_types=1);
```

Comment:
This looks minor, but I'm curious why its here now?

Reply:
I have no clue :sweat_smile: I am really struggling with this because the workflow keeps failing even though I have retried doing the schema stuff like 4 different times :confounded: